classes/class.tapestry.php

```diff
@@ -173,6 +174,11 @@ public function addNode($node)
             $this->rootId = $node->id;
         }
 
+        $nodeArray = [$node];
+        $nodeArray = $this->_addH5PMeta($nodeArray);
+        $node = array_values($nodeArray)[0];
+        $node = $tapestryNode->save($node);
+
```

Comment:
@odotadot we should not be saving h5p meta to the database because h5p meta might change outside of the tapestry context. This information needs to get passed to the frontend dynamically instead. I know this fix was put in to ensure that h5p meta is updated when a node is edited, but this is not the right way to do it. When a node is edited, we should return the new node information and dynamically retrieve and return the h5p meta for that node as well. The frontend should then take that information and update vuex with that.

Reply:
Removed those 4 lines of code. Everything still working fine (the bug that this code fixed is covered by other code in h5pForm.)